packages/diagrams/frontend/sirius-components-diagrams/src/renderer/edge/EdgeLayout.ts

```diff
@@ -86,8 +88,40 @@ const isVerticalLayoutDirection = (layoutDirection: string): boolean =>
   layoutDirection === 'DOWN' || layoutDirection === 'UP';
 const isHorizontalLayoutDirection = (layoutDirection: string): boolean =>
   layoutDirection === 'RIGHT' || layoutDirection === 'LEFT';
+const isPositionDependentRotation = (nodeData: NodeData): boolean => {
+  if ('positionDependentRotation' in nodeData) {
+    return (nodeData as FreeFormNodeData).positionDependentRotation;
+  }
+  return false;
+};
+const computeBorderNodeHandlePosition = (borderNodeData: NodeData, isInside: boolean): Position => {
+  let borderNodePosition: BorderNodePosition | null = borderNodeData.borderNodePosition;
+  if (isPositionDependentRotation(borderNodeData)) {
+    borderNodePosition = BorderNodePosition.WEST;
```

Comment:
Will this code be modified in the future when you'll handle the case of rotatable border nodes?

Reply:
This section calculates the correct handle position for a rotatable border node. So, in theory, it shouldn't be modified.